code/modules/atmospherics/machinery/portable/canister.dm

```diff
@@ -235,7 +235,7 @@
 		. += "can-open"
 	if(connected_port)
 		. += "can-connector"
-	var/pressure = air_contents.return_pressure()
+	var/pressure = air_contents?.return_pressure()
```

Comment:
this'll still runtime on the next line, since it returns null and you can't compare null to a number (that I know of)

Reply:
it didn't on my side
and it would only runtime if you try spawning the base object
also from the DM Doc 
>The operators >, <, >=, and <= test if the left-hand expression is greater than, less than, greater than or equal to, and less than or equal to the right-hand expression. These only apply to numerical expressions. Any other type of data (like null or a text string) is treated like 0.

Reply:
sure, but that's still a runtime that should be fixed